pkg/iam/iam.go

```diff
@@ -142,3 +163,112 @@ func (c *IAMClient) OrganizationIAM(ctx context.Context, organizationID string)
 
 	return m, nil
 }
+
+// RemoveProjectIAM removes the given IAM policy membership.
+func (c *IAMClient) RemoveProjectIAM(ctx context.Context, iamMember *assetinventory.AssetIAM) error {
+	// The Golang API does not support IAM removal on projects yet.
```

Comment:
I don't think there is API for removing IAM bindings. There is only API to set IAM policy. E.g. https://pkg.go.dev/cloud.google.com/go/resourcemanager/apiv3#ProjectsClient.SetIamPolicy

But I don't think it will necessarily make code simpler here.

Reply:
Yeah. That seems like it will be a lot more complicated and isn't as obvious IMO that we are removing a binding.

Reply:
In general, there are no APIs for updating bindings. Most tools, including gcloud, getIamPolicy to download the IAM policy document with an etag, they modify the policy document, then setIamPolicy to upload the document with the etag. If the server says there's a conflict, you retry. Example: https://github.com/GoogleCloudPlatform/berglas/blob/f69dc2be6e7d2f6647920b64d37adb1b88dc8d4d/pkg/berglas/iam_storage.go#L158-L195

Reply:
I updated it to use the cloudresourcemanager GetIAMPolicy and SetIAMPolicy. Specifically, I think this is nicer than gcloud because parsing out the retryable errors is significantly easier from the structured go error than from the stderr from a child process.